retroshare-gui/src/gui/NewsFeed.cpp

```diff
@@ -510,6 +520,40 @@ void NewsFeed::handleSecurityEvent(std::shared_ptr<const RsEvent> event)
 	NotifyQt::getInstance()->addToaster(RS_POPUP_CONNECT_ATTEMPT, e.mPgpId.toStdString().c_str(), det.location, e.mSslId.toStdString().c_str());
 }
 
+void NewsFeed::handleWireEvent(std::shared_ptr<const RsEvent> event)
+{
+    const RsWireEvent *pe =
+            dynamic_cast<const RsWireEvent*>(event.get());
+    if(!pe) return;
+
+    switch(pe->mWireEventCode)
+    {
+    case RsWireEventCode::NEW_WIRE:
+        addFeedItem( new WireNotifyGroupItem(this, NEWSFEED_WIRELIST, pe->mWireGroupId, false, true));
+        break;
+    case RsWireEventCode::NEW_POST:
+        addFeedItem( new WireNotifyGroupItem(this, NEWSFEED_WIRELIST, pe->mWireGroupId, false, true));
+        break;
+    case RsWireEventCode::POST_UPDATED:
```

Comment:
im wondering why you added for every event a addfeeItem thats to much :)
maybe you need only when a

NEW_WIRE (a new user joind to wire)
NEW_POST (user posted a tweet )
NEW_REPLY (reply comment to a post)
NEW_REPUBLISH ( a user republished my tweet)

this i think not needed as feed notify
NEW_LIKE ( maybe notify this only on wire button count and in wiredialog)
WIRE_UPDATED (this not needs notify as feeditem, its i think for internaly update the wire user list when im right)
POST_UPDATED (this not needs a notify as feeditem, it for update the posts when there is new posts or when im right)

Reply:
yea you are right i will only add for the 4 items you said.
the WIRE_UPDATED is the one where someone changes their profile photo or the tagline.
the POST_UPDATED is the one where someone updates their old post.